docs/reference/config-example.md

```diff
@@ -10,59 +10,81 @@ version: "3.7"
+          http:
+            port: 8080                                   # HTTP Liveness probe port. Only used when using an http probe type.
+            path: /status                                # HTTP Liveness probe path. Only used when using an http probe type.
+          ### TCP
+          type: tcp                                      # TCP Liveness probe type.
+          tcp:
+            port: 8080                                   # TCP Liveness probe port. Only used when using an tcp probe type.
+        failureThreshold: 3                              # Default: 3. Number of probe retries. Shared by all liveness probe types.
+          initialDelay: 1m0s                             # Default: 1m. How long to wait before initial probe run. Shared by all liveness probe types.
+          period: 1m0s                                   # Default: 1m. How often liveness probe should run for the workload. Shared by all liveness probe types.
+          timeout: 10s                                   # Default: 10s. Probe timeout. Shared by all liveness probe types.
+        readinessProbe:                                  # Workload's readiness probe
+          # EXER
+          type: exec                                     # Default: none. Possible options: none | exec | http | tcp.
+          exec:                                          # The exec command matching the readiness probe type.
+            command:                                     # Readiness probe command to run.
+              - echo
+              - not applicable
+          ### HTTP
+          type: http                                     # HTTP readiness probe type.
+          http:
+            port: 8080                                   # HTTP readiness probe port. Only used when using an http probe type.
+            path: /status                                # HTTP readiness probe path. Only used when using an http probe type.
+          ### TCP
+          type: tcp                                      # TCP readiness probe type.
+          tcp:
+            port: 8080                                   # TCP readiness probe port. Only used when using an tcp probe type.
+          failureThreshold: 3                            # Default: 3. Number of probe retries. Shared by all readiness probe types.
+          initialDelay: 1m0s                             # Default: 1m. How long to wait before initial probe run. Shared by all readiness probe types.
+          period: 1m0s                                   # Default: 1m. How often readiness probe should run for the workload. Shared by all readiness probe types.
+          timeout: 10s                                   # Default: 10s. Probe timeout. Shared by all readiness probe types.
+        replicas: 25                                     # Default: 1. Number of replicas per workload.
+        resource:                                        # Resource share request for a given workload
```

Comment:
Let's rephrase to `Defines resource requests and limits`? Could also reorder memory / maxMemory.